tests/frontend/travis/runnerBackend.sh

```diff
@@ -12,11 +12,8 @@ cd "${MY_DIR}/../../../"
 # Set soffice to /usr/bin/soffice
 sed 's#\"soffice\": null,#\"soffice\":\"/usr/bin/soffice\",#g' settings.json.template > settings.json.soffice
 
-# Set allowAnyoneToImport to true
-sed 's/\"allowAnyoneToImport\": false,/\"allowAnyoneToImport\": true,/g' settings.json.soffice > settings.json.allowImport
-
 # Set "max": 10 to 100 to not agressively rate limit
-sed 's/\"max\": 10/\"max\": 100/g' settings.json.allowImport > settings.json.rateLimit
+sed 's/\"max\": 10/\"max\": 100/g' settings.json.soffice > settings.json.rateLimit
```

Comment:
This looks wrong...  :??

Reply:
It's correct. The pattern for this file is:

  * `sed a > b`
  * `sed b > c`
  * `sed c > d`
  * etc.

I'm removing something from the middle of the chain:

  * `sed a > b`
  * `sed b > d`
  * etc.